projects/common/src/feature/state/feature-state-mock-provider.ts

```diff
@@ -0,0 +1,19 @@
+import { mockProvider } from '@ngneat/spectator/jest';
+import { FeatureStateResolver } from './feature-state.resolver';
+import { FeatureState } from './feature.state';
+import { Dictionary } from '../../utilities/types/types';
+import { FactoryProvider } from '@angular/core';
+import { of } from 'rxjs';
+
+type MockFeatureStateMap = Dictionary<FeatureState>;
+export const getMockFeatureStateResolver = (ffState: MockFeatureStateMap): FactoryProvider =>
```

Comment:
please move this into test utils project, not common.

Also curious on the need, can you ping me?

Reply:
+1